fe4u-boilerplate-main/src/css/style.css

```diff
@@ -0,0 +1,721 @@
+/* stylelint-disable */
+/*-------------------------NORMALIZE-----------------------*/
```

Comment:
Too long file. Please divide it into separate files:
- `normalize.css`
- `typography.css`. Only in `normalize.css` and `typography.css` you may set styles on `tag` level. In other places you may set styles only on `class` level.
- `favicon.css`
- `content.css` etc.